### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by JonasVervloet**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -20,19 +22,47 @@
 # Middleware to automatically open/close a database connection for every request
 @app.middleware('request')
 async def handle_request(request):
+    """Open database connection before each request is handled."""
```
Comment (juliavdkris) [inline src/main.py:28]: I personally think a # comment is a bit more readable than docstrings for single-line comments, but I don't know what PEP8 says

File: src/main.py [hunk 2]
```diff
@@ -20,19 +22,47 @@
...
+    Returns:
+    The response contains json name/value pairs 'success', 'updated' and 'message'.
+        'success' denotes if the right query parameters are available and if they are valid.
+        'updated' denotes if the webID uri and lblod uri pair is stored in the database.
+            This is set to False when the webID uri is already used in the database.
+        'message' clarifies the response.
+
+        Example:
+            {
+                'success': True
```
Comment (juliavdkris) [inline src/main.py]: Missing comma's here and using both single & double quotes in the same thing is kind of ugly

File: src/main.py [hunk 3]
```diff
@@ -101,6 +154,39 @@ async def get_handler(req):
 
 @app.route('/lists', methods=['GET'])
 async def get_handler(req):
+    """
+    Get all lists that are active for a given city.
+
```
Comment (juliavdkris) [inline src/main.py:168]: I'm being *really* pedantic here, but there's a double newline here while the other docstrings only have one

**Review by juliavdkris — APPROVED** [review]
Awesome! I had some small remarks about the formatting, but everything is super clear!

Also, I just noticed we're using snake_case and TitleCase interchangably, but this is probably not a good time to change that :p

File: src/main.py (hunk 1 above)
Comment (FlorentCollin) [reply src/main.py:28]: Apparently docstring is the preferred method and can be access directly in Python using the `__doc__`

File: src/main.py (hunk 2 above)
Comment (JonasVervloet) [reply src/main.py]: Which function is this?

Comment (juliavdkris) [reply src/main.py]: This is on the /store endpoint

**Review by juliavdkris — APPROVED** [review]

**Pull request merged**